app/helpers/application_helper.rb

```diff
@@ -43,6 +43,7 @@ def table_of_contents_separator(options = {})
     return if options[:value].blank?
     contents = options[:value][0].split('--').map(&:strip)
     contents = contents.join('<br />').html_safe # rubocop:disable Rails/OutputSafety
-    render partial: 'catalog/table_of_contents', locals: { contents: contents }
+    id = options[:document][:id]
```

Comment:
Is this a real `SolrDocument` instance? If so, you probably want to call the `#id` method instead of using hash-access.

Reply:
Thanks for the tip, @cbeer. I've updated my helper and spec accordingly.